.github/workflows/update_metadata.yml

```diff
@@ -7,31 +7,45 @@ on:
   workflow_dispatch:
 
 jobs:
-  generate:
+  update:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
+        with:
+          fetch-depth: 0  # Needed for git describe
+      
       - uses: dart-lang/setup-dart@v1
-
+      
       - name: Install dependencies
         run: dart pub get
-
+      
       - name: Fetch LibPhoneNumber Metadata
         run: curl -o resources/data_sources/PhoneNumberMetadata.xml https://raw.githubusercontent.com/google/libphonenumber/master/resources/PhoneNumberMetadata.xml
-
+      
       - name: Process Metadata
         run: dart resources/data_sources/convert_metadata.dart
-
+      
       - name: Generate Files
         run: |
           dart pub get
           dart resources/generate_files.dart && dart format lib/src && dart fix --apply
-
-      - name: Commit and Push Updated Files
+      
+      - name: Update Version and Create Tag
         run: |
+          # Read the current version from pubspec.yaml
+          CURRENT_VERSION=$(grep 'version:' pubspec.yaml | sed 's/version: //')
+          # Increment the patch version
+          NEW_VERSION=$(echo $CURRENT_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
```

Comment:
There is a dart command for this `dart pub bump patch`

Reply:
thanks!